FFWEB-1849: `idType` attribtue for ff-similar-products (#274)
FACT-Finder-Web-Components/documentation: markdown/3.x/en/api/ff-similar-products.api.md

| Name | Description |
| ---- | ----------- |
| **record-id**&nbsp;(Number) (default: empty) | The record ID of the product, for which similar products are returned. This also works for multiple products. In this case the record IDs have to be separated with `,`. |

> | **id-type**&nbsp;(String) **Options**: &nbsp;"id", &nbsp;"productNumber" (default: undefined) | The type of identifier for which the corresponding value should be passed in the **record-id**. This attribute is mandatory when using FACT-Finder NG. |

Reviewer: Honestly, I don't understand what this is trying to tell me.

Considering the FF API documentation
> Use this parameter to determine the type of id passed to the method.

Perhaps we can make something short and simple, too?

```suggestion
| **id-type**&nbsp;(String) **Options**: &nbsp;"id", &nbsp;"productNumber" (default: undefined) | This attribute determines the type of `record-id`. It is mandatory when using FACT-Finder NG. |
```

Author: Yes i had difficulties to to put it somehow neatly, this version is much better. Changed in v4 too
